site/quickstart.md

```diff
@@ -168,6 +164,8 @@ domain namespace (`sample-domain1-ns`), and the `domainHomeImageBase` (`oracle/w
 
 * Leaving the `image` empty unless you need to tag the new image that the script builds to a different name.
 
+**NOTE**: Make sure that your `JAVA_HOME` is set to a Java JDK version 1.8 or later.
```

Comment:
This is only needed for the WDT sample. By default, the WLST sample is used. Maybe we need to make this clear. 
How about adding "if you set the `domainHomeImageBuildPath` property to `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt`" to the end of this newly added sentence?

Reply:
In addition, the first bullet above refers to oracle/weblogic:12213-patch-wls-for-k8s. Shall we change it to store/oracle/weblogic:12.2.1.3? Hope this does not require retesting the quick start instructions.  Need to get Mark and Ryan's input on this.

Reply:
@doxiao Yes, all of these references should be set to store/oracle/weblogic:12.2.1.3.